src/utils/layer-utils.ts

```diff
@@ -0,0 +1,3 @@
+export function legendToStops(legend: { value: string; color: string }[] = []) {
```

Comment:
With this project, I'd say it's still up for debate whether it needs a global `utils` folder - I think until there end up being a lot of cross-dependencies, my preference is to keep the code close to where its used in mostly self-contained modules. To that end, I feel like this like file maybe should live in `MapView`?